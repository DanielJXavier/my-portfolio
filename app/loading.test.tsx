import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import Loading from "./loading";

describe("Loading page", () => {
  it("renders the page", () => {
    const { container } = render(<Loading />);

    expect(container).toMatchSnapshot();
  });
});
