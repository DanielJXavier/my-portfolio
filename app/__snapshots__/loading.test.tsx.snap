// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Loading page renders the page 1`] = `
<div>
  <main
    class="absolute top-0 right-0 bottom-0 left-0 flex items-center justify-center"
  >
    <div
      class="loading"
    >
      <p>
        dx
      </p>
    </div>
  </main>
</div>
`;
